gridstatus/nyiso.py

```diff
@@ -309,6 +316,32 @@ def get_lmp(
         df["Market"] = market.value
         df["Location Type"] = "Zone" if location_type == ZONE else "Generator"
 
+        # NYISO includes both RTD and RTC in the same file, so we have to differentiate
+        # them. The RTD values are at 5 minute intervals, while the RTC values are at
+        # 15 minute intervals (The first RTC value may be only 10 minutes after the last
+        # RTD value.)
+        if market in [Markets.REAL_TIME_5_MIN, Markets.REAL_TIME_15_MIN]:
+            df["Time Step"] = df["Interval Start"].diff(1)
+
+            # Find timestamps that are greater than 5 minutes
+            rtc_timestamps = df.loc[
+                df["Time Step"] > pd.Timedelta(minutes=5),
```

Comment:
Not sure how reliable this is but in the absence of more information, I think it's a reasonable assumption. The RTD values should always be five minutes apart.